src/index.js

```diff
@@ -1,66 +1,136 @@
 // import _ from 'lodash';
 import './style.css';
 
-const toDoList = [{
-  description: 'Complete project',
-  completed: 'false',
-  index: 0,
-},
-{
-  description: 'Read from books',
-  completed: 'false',
-  index: 1,
-},
-{
-  description: 'Do some sport',
-  completed: 'false',
-  index: 2,
-},
-];
-
 const toDoLlistDiv = document.querySelector('.to-do-list');
 const toDoListUl = document.createElement('ul');
-
 const toDoTitle = document.createElement('li');
 toDoTitle.innerHTML = "Today's To Do";
 toDoTitle.classList.add('task');
 toDoListUl.appendChild(toDoTitle);
 
-const toDoAdd = document.createElement('li');
-toDoAdd.innerHTML = 'Add to your list...';
+const toDoAdd = document.createElement('input');
+toDoAdd.type = 'text';
 toDoAdd.classList.add('task');
 toDoAdd.id = 'to-do-add';
-toDoAdd.placeholder = 'Add to your list';
+toDoAdd.placeholder = 'Add to your list...';
+toDoAdd.innerHTML = 'Add to your list...';
 
 toDoListUl.appendChild(toDoAdd);
-const tasksGenerate = () => {
-  for (let i = 0; i < toDoList.length; i += 1) {
+
+const toDoClear = document.createElement('li');
+toDoClear.innerHTML = 'Clear all completed';
+toDoClear.classList.add('task');
+toDoClear.id = 'to-do-clear';
+
+const toDoList = JSON.parse(localStorage.getItem('toDoList')) || [];
+let myIndex;
+const completed = false;
+
+class ToDoListTasks {
+  addTask = (description) => {
+    myIndex = toDoList.length;
```

Comment:
- [ ] [REQUIRED] Great implementation, However, let's tenure that the property index is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). Currently, your index starts from 0 instead of 1. Let's fix this as it is a core requirement.
![Screenshot from 2022-06-01 00-14-37](https://user-images.githubusercontent.com/58692590/171298863-1a76b92c-1d80-4716-8110-221d6f87eb44.png)